### PR TITLE
Add graphics export to wizard

### PR DESCRIPTION
**Pull request by robinschmid**

### CODE REVIEW

**Comment by robinschmid** [comment]
managed to fix graphics export in batch mode. This will call javafx thread and close it if initualized

**Comment by robinschmid** [comment]
Updated with the new exports

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/WorkflowDdaWizardParameters.java [hunk 1]
```diff
@@ -40,6 +40,8 @@ public final class WorkflowDdaWizardParameters extends WorkflowWizardParameters
       "Export for molecular networking (e.g., GNPS, FBMN, IIMN, MetGem)",
       "Export to Feature-based Molecular Networking (FBMN) and Ion Identity Molecular Networking (IIMN) on GNPS and other tools",
       true);
+  public static final BooleanParameter exportAnnotationGraphics = new BooleanParameter(
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/WorkflowDdaWizardParameters.java:43]: Do we want to enable the export by default? i don't think that's necessary

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/WorkflowDiaWizardParameters.java [hunk 2]
```diff
@@ -53,6 +53,8 @@ public final class WorkflowDiaWizardParameters extends WorkflowWizardParameters
       "Export for molecular networking (e.g., GNPS, FBMN, IIMN, MetGem)",
       "Export to Feature-based Molecular Networking (FBMN) and Ion Identity Molecular Networking (IIMN) on GNPS and other tools",
       true);
+  public static final BooleanParameter exportAnnotationGraphics = new BooleanParameter(
+      "Export annotation graphics", "Exports annotations to png and pdf images.", true);
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/WorkflowDiaWizardParameters.java:57]: same

File: src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/WorkflowDdaWizardParameters.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/modules/tools/batchwizard/subparameters/WorkflowDdaWizardParameters.java:43]: set all to false

**Pull request merged**